common/templates/recordset.html

```diff
@@ -12,6 +12,10 @@
                     <button id="search-submit" class="btn btn-primary" ng-click="search(vm.search)" role="button">
                         &nbsp;<span class="glyphicon glyphicon-search"></span>&nbsp;<!-- Added 2 non-breaking spaces to prevent btn's margins from collapsing and to center the icon-->
                     </button>
+                    <button type="button" tabIndex="-1" class="btn btn-primary btn-no-focus coltooltip">
+                        <span class="glyphicon glyphicon-question-sign"></span>
+                        <span class="coltooltiptext paragraph"><p>Use space to separate between conjunctive terms, | (no spaces) to separate disjunctive terms and quotations for exact phrases.</p><p>For example, <i><b>usc 1234</b></i> returns all records containing "usc" and "1234".</p><p><i><b>usc|1234</b></i> returns all records containing "usc" or "1234".</b></i></p><p><i><b>"usc 1234"</b></i> returns all records containing "usc 1234".</span>
```

Comment:
Some food for thought:
I debated for a while whether those `<b>` and `<i>` tags should be changed to `<strong>` and `<em>` tags respectively. Short answer is no, I think these are fine as is in this current case because the tooltip description here is just simply using bold and italic styles to visually separate the sample search terms from surrounding text — the search terms are not intrinsically more important than the surrounding text (like the way a paragraph heading would be).

Most people would say `<strong>` and `<em>` should be used instead because they're more "semantic", but [this article's details](http://stellify.net/html5-b-and-i-tags-are-going-to-be-useful-read-semantic-again/) on the W3C's HTML5 spec has me convinced `<b>` and `<i>` is correct here.